examples/nD_vectors_image.py

```diff
@@ -24,15 +24,13 @@
     # n x m grid of slanted lines
     # random data on the open interval (-1, 1)
     pos = np.zeros(shape=(m, n, p, 3), dtype=np.float32)
-    rand1 = 2 * (np.random.random_sample(n * m * p) - 0.5)
-    rand2 = 2 * (np.random.random_sample(n * m * p) - 0.5)
-
-    # assign projections for each vector
-    pos[:, :, :, 0] = 0
-    pos[:, :, :, 1] = rand1.reshape((m, n, p))
-    pos[:, :, :, 2] = rand2.reshape((m, n, p))
+    for i in range(3):
+        pos[:, :, :, i] = (
```

Comment:
usually we would just
```suggestion
        pos[..., i] = (
```
but it doesn't really matter

Reply:
fixed